packages/flutter/test/material/icons_test.dart

```diff
@@ -16,4 +16,28 @@ void main() {
     expect(Icons.clear.fontFamily, 'MaterialIcons');
     expect(Icons.search.fontFamily, 'MaterialIcons');
   });
+
+  testWidgets('Adaptive icons are correct on cupertino platforms',
+      (WidgetTester tester) async {
+    expect(Icons.adaptive.arrow_back, Icons.arrow_back_ios);
+    expect(Icons.adaptive.arrow_back_outlined, Icons.arrow_back_ios_outlined);
+  },
+      variant: const TargetPlatformVariant(<TargetPlatform>{
+        TargetPlatform.iOS,
+        TargetPlatform.macOS,
+      }));
```

Comment:
Indentation doesn't match the test below, might need a trailing comma or two

Reply:
Done